usbguard.te

```diff
@@ -0,0 +1,91 @@
+policy_module(usbguard, 0.1.0)
+
+########################################
+#
+# Declarations
+#
+
+## <desc>
+## <p>
+## Allow usbguard-daemon to write its config (e.g. /etc/usbguard/* without rules.conf)
+## Disabling this requires usbguard >0.7.0
+## </p>
+## </desc>
+gen_tunable(usbguard_daemon_write_conf, false)
+
+## <desc>
+## <p>
+## Allow usbguard-daemon to write rules (e.g. /etc/usbguard/rules.conf)
+## Enabled by default because a major usecase is to allow or reject
+## devices as a normal user with a desktop applet which talks to the
+## daemon with IPC.
+## </p>
+## </desc>
+gen_tunable(usbguard_daemon_write_rules, true)
+
+type usbguard_t;
+type usbguard_exec_t;
+init_daemon_domain(usbguard_t, usbguard_exec_t)
+
+type usbguard_unit_file_t;
+systemd_unit_file(usbguard_unit_file_t)
+
+type usbguard_conf_t;
+files_config_file(usbguard_conf_t)
+
+type usbguard_log_t;
+logging_log_file(usbguard_log_t)
+
+type usbguard_rules_t;
+files_config_file(usbguard_rules_t)
+
+type usbguard_tmpfs_t;
+files_tmpfs_file(usbguard_tmpfs_t)
+
+type usbguard_var_run_t;
+files_pid_file(usbguard_var_run_t)
+
+type usbguard_dbus_t;
```

Comment:
This is not needed, we can use only usbguard_t domain.

Reply:
@wrabcak if I remove usbguard_dbus_t (and the `dbus_system_domain(usbguard_dbus_t, usbguard_dbus_exec_t)` 

needed to re-add `dbus_system_domain(usbguard_t, usbguard_exec_t)` 

why not have a dedicated type for the dedicated usbguard-dbus daemon process?